doc/source/ray-air/config-scaling.rst

```diff
@@ -0,0 +1,36 @@
+.. _config-scaling:
+
+Configuring scaling
+===================
+
+Ths guide describes how you can use the ``ScalingConfig`` object configure resource utilization at the per-run level when training models with Ray AIR.
+
+Train Usage
+-----------
+
+To use ``ScalingConfig`` when training a model, pass in the ``scaling_config`` parameter to your ``Trainer``:
+
+.. literalinclude:: doc_code/config_scaling.py
+        :language: python
+        :start-after: __config_scaling_1__
+        :end-before: __config_scaling_1_end__
+
+
+Tune Usage
+----------
+
+You can also treat some scaling config variables as hyperparameters and optimize them using Tune. All Tune primitives are supported.
+
+If you use ``tune.grid_search``, then Tune will resolve the search space to yield variants for grid search variable. 
+
+For Tune primitives other than ``tune.grid_search``, no new variants will be generated, and the relevant variables will be sampled exactly once. 
```

Comment:
I think we can omit this since this is generic to Tune, and perhaps link to how Tune handles `grid_search` and other primitives.

Instead I think it's more important to call out that the user should populate a `ScalingConfig` into `scaling_config` of `param_space`, and how the user can insert `Domain`s into individual attributes of the `ScalingConfig` to have them treated as individual hyperparamters.